### PR TITLE
Janitorial cleanups

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/boot/bootctl.c [hunk 1]
```diff
@@ -923,30 +923,29 @@ static bool same_entry(uint16_t id, sd_id128_t uuid, const char *path) {
 
 static int find_slot(sd_id128_t uuid, const char *path, uint16_t *id) {
         _cleanup_free_ uint16_t *options = NULL;
-        int n, i;
 
-        n = efi_get_boot_options(&options);
+        int n = efi_get_boot_options(&options);
```
Comment (medhefgo) [inline src/boot/bootctl.c:927]: commit message: What are "interator"s? :P

File: src/shared/condition.c [hunk 2]
```diff
@@ -212,7 +207,7 @@ static bool test_order(int k, OrderOperator p) {
 static int condition_test_kernel_version(Condition *c, char **env) {
         OrderOperator order;
         struct utsname u;
-        const char *p;
+        ;
```
Comment (medhefgo) [inline src/shared/condition.c]: Single line `;`

File: src/boot/bootctl.c (hunk 1 above)
Comment (keszybz) [reply src/boot/bootctl.c:927]: Special C magic ;)

File: src/shared/condition.c (hunk 2 above)
Comment (keszybz) [reply src/shared/condition.c]: Both fixed.

**Review by medhefgo — APPROVED** [review]

**Review by bluca — APPROVED** [review]

**Pull request merged**